draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -69,11 +69,18 @@ signed JWT might want to disclose to Verifiers at some point. The
 ability to selectively disclose a subset of these claims depending on
 the Verifier becomes crucial to ensure minimum disclosure and prevent
 Verifiers from obtaining claims irrelevant for the transaction at hand.
-
 One example of such a multi-use JWT is a verifiable credential, a
 tamper-evident credential with a cryptographically verifiable authorship that
-contains claims about a subject. SD-JWTs defined in this document enable such
-selective disclosure of claims.
+contains claims about a subject.
+
+SD-JWTs defined in this document enable such selective disclosure of claims for JWTs.
+
+Similar to the JWT specification on which it builds, this document is a product of the
+Web Authorization Protocol (oauth) working group. However, while both JWT and SD-JWT
+have potential OAuth 2.0 applications, their utility and application is certainly not constrained to OAuth 2.0.
+JWT was developed as a general-purpose token format and has seen widespread usage in a
+variety of applications. SD-JWT is a selective disclosure mechanism for JWT and is
+similarly intended to be general-purpose specification.
```

Comment:
Shall we add something like "... and this is why this specification in general does not use OAuth-specific terms like client and authorization server but instead more broadly applicable terms like Verifier and Holder"?

Reply:
I know that was part of Hannes' question but I think that's more than should be said here.

Reply:
i.e. I wanted to give some context for why this thing is coming out of the OAuth WG and state the general-purpose intent. But don't want to try and touch on each question/objection from the particular email from Hannes and don't want to get into specifics of how SD-JWT would be used w/ OAuth.

Reply:
That works for me!